app/models/accounting/transaction.rb

```diff
@@ -0,0 +1,37 @@
+# == Schema Information
+#
+# Table name: accounting_transactions
+#
+#  created_at          :datetime         not null
+#  id                  :integer          not null, primary key
+#  qb_transaction_id   :string           not null
+#  qb_transaction_type :string           not null
+#  updated_at          :datetime         not null
+#
+
+class Accounting::Transaction < ActiveRecord::Base
+  attr_writer :qb_object
```

Comment:
I would normally have this private but because it needs to be set externally, I have I public.